.tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "noEmit": true,
+    "module": "commonjs",
+    "target": "es2018",
+    "lib": ["es2018"],
+    "allowJs": true,
+    "checkJs": true,
+    "strict": true,
+    "resolveJsonModule": true,
+    "noImplicitAny": false
+  },
+  "include": [
+    "lib/css-tools.js",
```

Comment:
shouldn't this be a wildcard like `"lib/**/*.js`?

Reply:
Ideally yes, but the files have many issues and doing it all at once won't be possible. That's why I preferred to only add those files that were fixable with minimum changes.

Also, when you include one file it will also check all the transitive dependencies, so including any of the svgo entrypoints or config, will bring all the plugins as well.